tests/functional/object/mcg/test_noobaa_endpoint_keda.py

```
@@ -0,0 +1,235 @@
+        logger.info("Waiting for ScaledObject 'noobaa' to be created")
+        try:
+            for exists in TimeoutSampler(120, 10, _scaled_object_exists):
+                if exists:
+                    logger.info("ScaledObject created by noobaa-operator")
+                    break
+        except TimeoutExpiredError:
+            raise TimeoutExpiredError("ScaledObject was not created")
+
+        hpa_ocp = OCP(kind="HorizontalPodAutoscaler", namespace=namespace)
+
+        def _hpa_deleted():
```

Ruff (0.15.20)

[warning] Missing return type annotation for private function `_hpa_deleted`

Add return type annotation: `bool | None`

(ANN202)